### PR TITLE
Whitespace: fix some confusing mixes of tabs and spaces

### PR DESCRIPTION
**Pull request by vivid-synth**
Clean up a few ambiguous mixes of spaces and tabs. Note this is conservative: mixes for layout, like:

```
\tfoo(2
\t    5);
```

have not been removed.

### CODE REVIEW

File: server/scsynth/SC_ComPort.cpp [hunk 1]
```diff
@@ -328,18 +328,18 @@ class SC_TcpConnection:
...
 		namespace ba = boost::asio;
 		async_read(socket, ba::buffer(&OSCMsgLength, sizeof(OSCMsgLength)),
-				   boost::bind(&SC_TcpConnection::handleLengthReceived, shared_from_this(),
-							   ba::placeholders::error,
-							   ba::placeholders::bytes_transferred)
-				   );
+		           boost::bind(&SC_TcpConnection::handleLengthReceived, shared_from_this(),
+		           ba::placeholders::error,
+		           ba::placeholders::bytes_transferred)
+		          );
```
Comment (brianlheim) [inline server/scsynth/SC_ComPort.cpp]: I think you messed up the alignment on this one. Lines 332 and 333 should be aligned with the opening paren of line 331. Maybe the inner close paren should be moved to its own line as well.

File: server/scsynth/SC_ComPort.cpp [hunk 2]
```diff
@@ -356,13 +356,13 @@ class SC_TcpConnection:
 		data = (char*)malloc(OSCMsgLength);
 
 		async_read(socket, ba::buffer(data, OSCMsgLength),
-				   boost::bind(&SC_TcpConnection::handleMsgReceived, shared_from_this(),
-							   ba::placeholders::error,
-							   ba::placeholders::bytes_transferred));
+		           boost::bind(&SC_TcpConnection::handleMsgReceived, shared_from_this(),
+		           ba::placeholders::error,
+		           ba::placeholders::bytes_transferred));
```
Comment (brianlheim) [inline server/scsynth/SC_ComPort.cpp]: Same as lines 331-334 above. The outer close paren should also be moved for consistency (while I've got my pedantry cap on)

File: server/scsynth/SC_ComPort.cpp [hunk 3]
```diff
@@ -429,13 +429,13 @@ class SC_TcpInPort
 			SC_TcpConnection::pointer newConnection (new SC_TcpConnection(mWorld, ioService, this));
 
 			acceptor.async_accept(newConnection->socket,
-								  boost::bind(&SC_TcpInPort::handleAccept, this, newConnection,
-											  boost::asio::placeholders::error));
+			                      boost::bind(&SC_TcpInPort::handleAccept, this, newConnection,
+			                      boost::asio::placeholders::error));
```
Comment (brianlheim) [inline server/scsynth/SC_ComPort.cpp]: Same as lines 331-334.

File: server/scsynth/SC_CoreAudio.h [hunk 4]
```diff
@@ -354,22 +354,22 @@ class SC_AndroidJNIAudioDriver : public SC_AudioDriver
...
 protected:
 
-    // Driver interface methods
-        virtual bool DriverSetup(int* outNumSamplesPerCallback, double* outSampleRate);
-        virtual bool DriverStart();
-        virtual bool DriverStop();
+	// Driver interface methods
+		virtual bool DriverSetup(int* outNumSamplesPerCallback, double* outSampleRate);
+		virtual bool DriverStart();
+		virtual bool DriverStop();
```
Comment (brianlheim) [inline server/scsynth/SC_CoreAudio.h]: Shouldn't these three lines (358-360) just have one indent? (See 319-321 above)

File: server/scsynth/SC_CoreAudio.h [hunk 5]
```diff
@@ -354,22 +354,22 @@ class SC_AndroidJNIAudioDriver : public SC_AudioDriver
...
         
 public:
-    SC_AndroidJNIAudioDriver(struct World *inWorld);
-        virtual ~SC_AndroidJNIAudioDriver();
+	SC_AndroidJNIAudioDriver(struct World *inWorld);
+		virtual ~SC_AndroidJNIAudioDriver();
 
-        void genaudio(short* arri, int numSamples);
+		void genaudio(short* arri, int numSamples);
```
Comment (brianlheim) [inline server/scsynth/SC_CoreAudio.h]: Compare to 324-327 above; I think these should just have one level of indent.

File: server/scsynth/SC_SequencedCommand.cpp [hunk 6]
```diff
@@ -1146,7 +1146,7 @@ bool BufCloseCmd::Stage2()
 {
 #ifdef NO_LIBSNDFILE
 	SendFailure(&mReplyAddress, "/b_close", "scsynth compiled without libsndfile\n");
- 	scprintf("scsynth compiled without libsndfile\n");
+	scprintf("scsynth compiled without libsndfile\n");
```
Comment (brianlheim) [inline server/scsynth/SC_SequencedCommand.cpp]: Side note--I smell a possible refactor here, especially with the repeated `#ifdef`s.

**Comment by vivid-synth** [comment]
Thanks @brianlheim - those are exactly the kinds of wrong-indentation-can-imply-the-wrong-meaning problems I was trying to fix!

**Comment by brianlheim** [comment]
Thanks for this--every little bit helps!

**Pull request merged**